dist/src/main/config/broker.standalone.yaml.template

```diff
@@ -682,6 +682,20 @@
         # This setting can also be overridden using the environment ZEEBE_BROKER_BACKPRESSURE_GRADIENT2_LONGWINDOW
         # longWindow: 600
 
+    # exporting:
+      # This section contains definitions common to all exporters.
+      #
+      # Skip records Exporter ------------
```

Comment:
:x: This appears to be left-over. There is no "Skip records Exporter". The following lines should be indented one level less.